src/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+  </head>
+  <body>
+    <div class="container">
+      <!-- Header -->
+      <header>
+        <a href="">
+          <img
+            src="https://placehold.co/40x40/orange/white?font=Lora&text=YAS"
+            alt=""
+          >
+          <span>Your Amazing Shows</span>
+        </a>
+      </header>
+      <!-- Main-->
+      <div id="mainSection" class="main"></div>
+      <!-- Dialog -->
+      <dialog class="modal" id="modal">
+        <span class="modal-close">x</span>
+        <div class="card-image">
+          <img src="https://placehold.co/400x400/" id="card-image" alt="">
+        </div>
+        <div class="modal-body">
+          <h3 id="modal-name">name</h3>
+          <p id="modal-description"></p>
+          <h4>Comments <span id="comment-number"></span></h4>
```

Comment:
- Kindly, you missed here also showing the number of comments given for that specific movie, so please add a count function that counts the number of comments and displays it, and doesn't use API response. length to show the number of comment Lists because you will use this function later for testing :+1: